pydis_site/apps/content/resources/guides/pydis-guides/contributing/forking-repository.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Forking a Repository
+description: A guide to forking git repositories.
+icon: fab fa-github
+---
+
+Before contributing to any project, you will have to fork the project, ie. create your own online copy of the project.
+The following will use the [Sir-Lancebot](https://github.com/python-discord/sir-lancebot/) repository as an example, but the steps are the same for all other repositories.
+
+1. Navigate to the repository page and press the `Fork` button at the top of the page.
+![Github Fork Button](/static/images/content/contributing/fork_button.webp)
+2. Fork it to your account.  
+![Github Fork to User](/static/images/content/contributing/fork_user.webp)
+3. Later, you will need the Git URL of your forked repository in order to clone it.
+In your newly forked repository, copy the Git URL by clicking the green `Code` button, then click the Copy Link button.
+![Github Fork Clone URL](/static/images/content/contributing/fork_clone.webp)
```

Comment:
This would benefit from some extra padding between the text and the images.

Reply:
This actually applies to all images in guides, but I don't consider it too important.

Reply:
I found that it only applies to images in lists, as in the other cases it would be separated by paragraph padding.

One solution was to add even more `<br>` tags (we can't put a newline in the markdown otherwise the image would extend past the list item's left margin), but I've decided to use CSS to avoid having article writers think about it.